src/store/modules/order/OrderState.ts

```diff
@@ -3,9 +3,5 @@ export default interface OrderState {
     list: any;
     total: number;
   },
-  current: {
```

Comment:
Unwanted change. Please revert

Reply:
We defined the order state this way because it was throwing error : Argument of type 'any' is not assignable to parameter of type 'never'.